Assets/Mirror/Runtime/Transport/Transport.cs

```diff
@@ -125,6 +126,21 @@ public virtual bool Available()
         /// <returns>true if the data was sent</returns>
         public abstract bool ServerSend(int connectionId, int channelId, byte[] data);
 
+        /// <summary>
+        /// Send data to multiple clients at once.
+        /// Calls ServerSend by default. Can be overwritten by transports that can send to multiple
+        /// clients at once more efficiently if needed.
+        /// </summary>
+        public virtual bool ServerSend(List<int> connectionIds, int channelId, byte[] data)
```

Comment:
You cannot call this method without allocating a byte[]

You cannot cache byte [] because they need the exact length. A byte[] of size 100 will not work for a message of size 99.

If the goal is to do allocation free sends, you must pass an arraysegment or a message